.azure-pipelines/templates/automation_test.yml

```diff
@@ -7,6 +7,9 @@ parameters:
 - name: profile
   type: string
   default: latest
+- name: instance
+  type: string
+  default: '1_1'
```

Comment:
What does `1_1` mean?

Reply:
1_1 mean 1 instance, so we don't automatic shedule test modules or raise error when not specify instance.

Reply:
Is there any document explaining this? otherwise it's hard to maintain.